crates/egui/src/context.rs

```diff
@@ -599,6 +599,14 @@ impl ContextImpl {
             .unwrap_or(&ViewportId::ROOT)
     }
 
+    /// Return the `ViewportId` of his parent.
+    pub(crate) fn get_parent_viewport_id(&self, viewport_id: ViewportId) -> ViewportId {
```

Comment:
https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter


```suggestion
    pub(crate) fn parent_viewport_id_of(&self, viewport_id: ViewportId) -> ViewportId {
```

Reply:
> https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter



> https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

This name does not pass inspection. Should we use a different name?

Reply:
> https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

what do you want?

parent_viewport_id_of() ? (you're writed here)
viewport_parent_id_of() ? (you're writed title)